python-backend/app/api/variances/models/variance.py

```diff
@@ -45,6 +45,10 @@ class Variance(AuditMixin, Base):
     mine_documents = db.relationship(
         'MineDocument', lazy='joined', secondary='variance_document_xref')
     inspector = db.relationship('Party', lazy='joined', foreign_keys=[inspector_party_guid])
+    mine = db.relationship('Mine', lazy='joined')
+
+    mine_name = association_proxy('mine', 'mine_name')
```

Comment:
I don't love having `mine_name` on the variance response, but I can see why it might be necessary in practice. What was the reason for needing to add it?

Reply:
When viewing the table of variances sitewide, it is necessary to know what mine the variance belongs to, so we're displaying the `mine_name` and a link to the mine on the table view of variances.

Reply:
In-theory, that's what the `mine_guid` is for, but I can see why making a separate API call to fetch the mine name isn't ideal.

I think it might be worth coming up with a pattern for conditionally requesting these related fields. Similar to how we are/were treating nested response objects. But that's a larger team discussion. Marking this as resolved.